### ISSUE TITLE
arrow heads not showing on edges

### ISSUE THREAD

**Issue by bgriffen** [oxfordinternetinstitute/gephi-plugins#62]
I added 

```
  "defaultEdgeArrow": "target"
```

to my `config.json` file and added the `"arrow":"source"` attribute to my `edges` component of the data.json file but they still don't appear in my graph. 

```
{"target": "37", "color": "rgb(0,109,44)", "source": "36", "attributes": {},"arrow":"source", "id": "2466", "size": 108.0}
```

Any help would be great. There is some discussion [here](https://github.com/jacomyal/sigma.js/issues/26) and [here](https://stackoverflow.com/questions/21841910/directed-edges-in-sigma-js-a-minimal-example) but there seems to be no solution. I ideally would like this to work for straight or curved edges.

**Comment by bgriffen** [comment]
Anyone able to take a look at this form me please? Thanks!

**Comment by computermacgyver** [comment]
Hi bgriffen. The plugin uses the original 0.8 version of sigma at the moment. Arrows do not appear on curved edges, but do appear on straight edges if you change defaultEdgeType to line in the config.json file
`"defaultEdgeType": "line",`
If that doesn't work, I can dig up my modifications to sigma to add arrows to straight edges or you can use the newer (1.0) version of sigma (but then not this plugin). A newer version of the plugin with the new version of sigma is in the works and once this is out edge arrows will be easier to add, but this will take a bit of time.

**Comment by bgriffen** [comment]
Thanks though even when I set `defaultEdgeType` to `line`, they still don't appear with arrows. Is there something else I need to set?

![Example image.](https://i.imgur.com/PUHthob.png)

**Comment by rosemorozenkova** [comment]
> Thanks though even when I set defaultEdgeType to line, they still don't appear with arrows. Is there something else I need to set?

I have the same problem. Even if to use updated versions of Sigma.js and Gephi. Result is the same with all options:  `line`, `curvedArrow`, `arrow` and `curve`.

Please, help.

**Comment by schignel** [comment]
I am having the same problem as @bgriffen and @RoseAysina. Has anyone managed to get arrows to appear on the exported graph?

**Comment by computermacgyver** [comment]
Looks like I [added this to the GitHub repo](https://github.com/oxfordinternetinstitute/InteractiveVis/commit/72d85981dc5c1343c16f42387d650b78bf179592) 2 years ago, but didn't ever repackage the plugin to include it. 

You'll have to update the index.html file, add js/canvasutilities.js, and replace js/sigma/sigma.min.js as per [this commit](https://github.com/oxfordinternetinstitute/InteractiveVis/commit/72d85981dc5c1343c16f42387d650b78bf179592). 

In addition, you will need to open your config.json file in a text editor (e.g., notepad) and add/change "defaultEdgeType" to "arrow".

Alternatively, you can use the latest version of Sigma.js directly without the HTML/JS in this repository, which is for an older version.

**Comment by schignel** [comment]
Hi @computermacgyver, I believe I've followed your instructions and am still not getting any arrows in my graph. I've attached my index and config files here. 

Here is it in Gephi:
![image](https://user-images.githubusercontent.com/19474876/34252819-ffc5dec8-e6a9-11e7-850a-f81f07a934a4.png)

And here in the Sigma export:
![image](https://user-images.githubusercontent.com/19474876/34252794-ec0db5f4-e6a9-11e7-9778-22ed7d555375.png)

I am very new to sigma.js and web development in general, so I apologize if I'm missing something obvious. 

[config and index.zip](https://github.com/oxfordinternetinstitute/gephi-plugins/files/1578917/config.and.index.zip)
